e2e/test/scenarios/admin/performance/schedule.cy.spec.ts

```diff
@@ -0,0 +1,163 @@
+    cy.signInAsAdmin();
+    setTokenFeatures("all");
+  });
+
+  /** An object describing the values to enter in the schedule strategy configuration form. */
+  type FormValues = Partial<Record<ScheduleComponentType, string>>;
+
+  type ScheduleTestOptions = [
+    FormValues,
+    /** The cron expression we expect to be sent to the API. */
+    string,
+  ];
+
+  // prettier-ignore
+  const schedules: ScheduleTestOptions[] = [
+    [{}, "0 0 * * * ?"], // The default schedule is 'every hour, on the hour'
+
+    [{ frequency: "daily" }, "0 0 8 * * ?"], // 8 am is the default time for a daily schedule
+
+    [{ frequency: "hourly" }, "0 0 * * * ?"], // Check that switching back to hourly works
+
+    [{ frequency: "daily", time: "9:00" }, "0 0 9 * * ?"], // AM is the default choice between AM and PM
+    [{ frequency: "daily", time: "9:00", amPm: "PM" }, "0 0 21 * * ?"],
+    [{ frequency: "daily", time: "12:00", amPm: "AM" }, "0 0 0 * * ?"],
+
+    [ { frequency: "weekly", weekday: "Monday", time: "12:00", amPm: "AM" }, "0 0 0 ? * 2" ],
+    [ { frequency: "weekly", weekday: "Monday", time: "1:00", amPm: "AM" }, "0 0 1 ? * 2" ],
+
+    [ { frequency: "weekly", weekday: "Tuesday", time: "2:00", amPm: "AM" }, "0 0 2 ? * 3" ],
+    [ { frequency: "weekly", weekday: "Tuesday", time: "3:00", amPm: "AM" }, "0 0 3 ? * 3" ],
+
+    [ { frequency: "weekly", weekday: "Wednesday", time: "4:00", amPm: "AM" }, "0 0 4 ? * 4" ],
+    [ { frequency: "weekly", weekday: "Wednesday", time: "5:00", amPm: "AM" }, "0 0 5 ? * 4" ],
+
+    [ { frequency: "weekly", weekday: "Thursday", time: "6:00", amPm: "AM" }, "0 0 6 ? * 5" ],
+    [ { frequency: "weekly", weekday: "Thursday", time: "7:00", amPm: "AM" }, "0 0 7 ? * 5" ],
+
+    [ { frequency: "weekly", weekday: "Friday", time: "8:00", amPm: "AM" }, "0 0 8 ? * 6" ],
+    [ { frequency: "weekly", weekday: "Friday", time: "9:00", amPm: "AM" }, "0 0 9 ? * 6" ],
+
+    [ { frequency: "weekly", weekday: "Saturday", time: "10:00", amPm: "AM" }, "0 0 10 ? * 7" ],
+    [ { frequency: "weekly", weekday: "Saturday", time: "11:00", amPm: "AM" }, "0 0 11 ? * 7" ],
+
+    [ { frequency: "weekly", weekday: "Sunday", time: "12:00", amPm: "PM" }, "0 0 12 ? * 1" ],
+    [ { frequency: "weekly", weekday: "Sunday", time: "1:00", amPm: "PM" }, "0 0 13 ? * 1" ],
+
+    [ { frequency: "monthly", frame: "first", weekdayOfMonth: "Sunday", time: "12:00", amPm: "AM" }, "0 0 0 ? * 1#1" ],
+    [ { frequency: "monthly", frame: "first", weekdayOfMonth: "Monday", time: "2:00", amPm: "AM" }, "0 0 2 ? * 2#1" ],
+
+    [ { frequency: "monthly", frame: "last", weekdayOfMonth: "Tuesday", time: "12:00", amPm: "AM" }, "0 0 0 ? * 3L" ],
+    [ { frequency: "monthly", frame: "last", weekdayOfMonth: "Wednesday", time: "12:00", amPm: "AM" }, "0 0 0 ? * 4L" ],
+
+    [ { frequency: "monthly", frame: "15th", time: "12:00", amPm: "AM" }, "0 0 0 15 * ?" ],
+    [ { frequency: "monthly", frame: "15th", time: "11:00", amPm: "PM" }, "0 0 23 15 * ?" ],
+
+  ];
```

Comment:
I'll let you be the judge but keep in mind that with e2e tests:
- they take a long time
- so we try to use them sparingly to cover complex UI cases
- that can't easily be covered by much faster unit tests

You know this code better than I do, but it looks to me that we're exercising most of this functionality multiple times, and some of the more granular testing can be handled in unit tests. (e.g. it's probably not worth it to have a test for every day of the week, probably a single hourly/daily/weekly/monthly test is enough for e2e, and you can drill down in more detail in unit tests)